src/components/Map/LayerControl.tsx

```diff
@@ -19,12 +18,14 @@ type Props = {
   onMarkerClick: (_event: any, _markerData: ChannelData) => void;
 };
 
+const getIntensity = (data: ChannelData): number => data.intensity;
+
 const useHeatmapPoints = (locations: ChannelData[]) => {
   return useMemo(
     () =>
       locations.map(
         (item) =>
-          [item.location.lng, item.location.lat, DEFAULT_IMPORTANCY] as Point
+          [item.location.lng, item.location.lat, getIntensity(item)] as Point
```

Comment:
Why we didn't get intensity directly as `item.intensity`?

Reply:
good q, i started with having a helper idea, then kept it that way. you're right we can just use `item.intensity` here.

Reply:
i'll do this in a follow up pr